app/main.ts

```diff
@@ -166,6 +166,32 @@ function sendGameData(gameData: GameData): void {
   );
 }
 
+// Win condition
+
+function canPawnMove(pawn: Pawn): boolean {
+  if (!(gameData?.positionPawn == undefined)) {
+    let flag = false;
+    for (const pawnAdjacent of pawn.adjacent) {
+      const b = boxes[pawnAdjacent[1]][pawnAdjacent[0]];
+      if (!b.removed && !equalArray(pawnAdjacent, gameData.positionPawn[0]) && !equalArray(pawnAdjacent, gameData.positionPawn[1])) {
+        flag = true;
+      }
+    }
+    return flag;
+  }
+  return true;
+}
+
+function getWinCondition(): WinCondition {
+  if (!canPawnMove(pawns[0]) && gameData.turn === Turn.PLAYER_0) {
+    return WinCondition.PLAYER_1_WON;
+  } else if (!canPawnMove(pawns[1]) && gameData.turn === Turn.PLAYER_1) {
+    return WinCondition.PLAYER_0_WON;
+  } else {
+    return WinCondition.FALSE;
+  }
```

Comment:
`else if` or `else` can be removed when you `return`
```suggestion
  if (!canPawnMove(pawns[0]) && gameData.turn === Turn.PLAYER_0) {
    return WinCondition.PLAYER_1_WON;
  }
  
  if (!canPawnMove(pawns[1]) && gameData.turn === Turn.PLAYER_1) {
    return WinCondition.PLAYER_0_WON;
  }
  
  return WinCondition.FALSE;
```